webpack.config.js

```diff
@@ -43,40 +43,31 @@ const distAssets = path.resolve(__dirname, process.env.npm_package_config_distAs
 // Start configuring webpack.
 var webpackConfig = {
   // What am i?
-  name: 'stanford_basic',
+  name: 'soe_basic',
   // Allows for map files.
   devtool: 'source-map',
   // What build?
   entry: {
-    "base": path.resolve(__dirname, srcJS + "/base.js"),
-    "behaviors": path.resolve(__dirname, srcJS + "/behaviors.js"),
-    "components": path.resolve(__dirname, srcSass + "/components/index.scss"),
-    "layout": path.resolve(__dirname, srcSass + "/layout/index.scss"),
-    "print": path.resolve(__dirname, srcSass + "/print/index.scss"),
-    "state": path.resolve(__dirname, srcSass + "/state/index.scss"),
-    "theme": path.resolve(__dirname, srcSass + "/theme/index.scss"),
-    "ckeditor": path.resolve(__dirname, srcSass + "/ckeditor.scss"),
+    "base":          path.resolve(srcSass, "base/index.scss"),
+    "scripts":       path.resolve(srcJS,   "scripts.js"),
+    "components":    path.resolve(srcSass, "components/index.scss"),
+    "layout":        path.resolve(srcSass, "layout/index.scss"),
+    "print":         path.resolve(srcSass, "print/index.scss"),
+    "state":         path.resolve(srcSass, "state/index.scss"),
+    "theme":         path.resolve(srcSass, "theme/index.scss"),
   },
   // Where put build?
   output: {
     filename: "[name].js",
-    path: path.resolve(__dirname, distJS)
-  },
-  // Relative output paths for css assets.
-  resolve: {
-    alias: {
-      'decanter-assets': path.resolve(npmPackage + 'decanter/core/src/img'),
-      'decanter-src': path.resolve(npmPackage + 'decanter/core/src'),
-      '@fortawesome': path.resolve(npmPackage + '@fortawesome'),
-    }
+    path: path.resolve(distJS)
   },
   // Additional module rules.
   module: {
     rules: [
       // Drupal behaviors need special handling with webpack.
       // https://www.npmjs.com/package/drupal-behaviors-loader
       {
-        test: /\.behavior.js$/,
+        // test: /\.behavior.js$/,
```

Comment:
@katrialesser Why is this commented out? This is needed for the drupal behavior fixes FWIK.

Reply:
Oh sorry! That was when I was debugging why the build wasn't working, changing back!